service/history/timer_queue_standby_task_executor.go

```diff
@@ -166,7 +160,7 @@ func (t *timerQueueStandbyTaskExecutor) executeUserTimerTimeoutTask(
 				timerTask.GetVisibilityTime(),
 				timerSequenceID.Timestamp,
 			) {
-				return getHistoryResendInfo(mutableState)
+				return &struct{}{}, nil
```

Comment:
nit: seem like we don't need extra data in the struct. could we just use a boolean?

Reply:
Current interface is taking an struct as input. I would love to keep the interface as is for now.